.gitignore

```diff
@@ -33,6 +33,7 @@ firebase-debug.log
 
 firebase-sw-private.js
 /.firebase
+.firebaserc
```

Comment:
What this means? You committed that file and gitignore the file same time? Any specific reason?

Reply:
I thought that it will differ for every member that project id , so i thought it would be better to use env varible or put it in gitignore . I removed firebaserc from gitignore , now there will be no problem